### PR TITLE
rework image article and upgrade versions

### PR DESCRIPTION
**Pull request by nakata5321**
Update article "Pre-installed Image For Raspberry Pi" with autogenerated image and update tested versions in articles.

### CODE REVIEW

File: docs/en/hass-install-image.md [hunk 1]
```diff
@@ -62,37 +59,39 @@ Select required image as the operating system and ensure to select your SD card
...
-    optional: true
-wifis:
-  wlan0:
-    dhcp4: true
-    optional: true
-    access-points:
-      YOUR_WIFI_NAME:
-        password: "YOUR_WIFI_PASSWORD"
+Inside this folder you need to create a `wpa_supplicant.conf` file, where you will provide your network credentials.
+Parse to the file next:
```
Comment (PaTara43) [inline docs/en/hass-install-image.md:65]: Paste the following to the file. Parse - не то чутка)

File: docs/en/hass-install-image.md [hunk 2]
```diff
@@ -62,37 +59,39 @@ Select required image as the operating system and ensure to select your SD card
...
+        scan_ssid=1
+        ssid="<Name of your wireless LAN>"
+        psk="<Password for your wireless LAN>"
+        proto=RSN
+        key_mgmt=WPA-PSK
+        pairwise=CCMP
+        auth_alg=OPEN
+}
 ```
+And change fields `ssid` and `psk` to your Wi-Fi name and password. Also write your country to the `country` field. Coutry code you can find [here](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes).
```
Comment (PaTara43) [inline docs/en/hass-install-image.md:81]: Можно set ssid, psk and country code according to yours.  You can find country codes here.

File: docs/en/hass-install-image.md [hunk 3]
```diff
@@ -62,37 +59,39 @@ Select required image as the operating system and ensure to select your SD card
...
 
 <robo-wiki-note type="warning">
-
   Make sure that you input your actual Wi-Fi name and your Wi-Fi password.
-
 </robo-wiki-note>
 
-<robo-wiki-note type="note">This Wi-Fi settings available only on first boot setup.
-If later you need to change settings, please edit configuration file in `/etc/netplan/` folder. 
+<robo-wiki-note type="note"> This Wi-Fi settings available only on first boot setup.
```
Comment (PaTara43) [inline docs/en/hass-install-image.md:87]: This way of setting Wi-Fi credentials is only available before the first boot. If you need to change it later, please, use...
Давно хотел переделать предложение аххааа

File: docs/en/hass-install-image.md [hunk 4]
```diff
@@ -62,37 +59,39 @@ Select required image as the operating system and ensure to select your SD card
...
+If later you need to change settings, please use `sudo raspi-config` command. 
 </robo-wiki-note>
 
 Then you need to save the file, **safely eject the SD card**, and insert it into the Raspberry Pi and turn it on. It should connect to your wi-fi network. 
 
+> After your Raspberry Pi is connected to power, make sure to wait a few (up to 5) minutes for it to boot up and register on the network.
```
Comment (PaTara43) [inline docs/en/hass-install-image.md:93]: Raspberry Pi is plugged in...

**Review by PaTara43 — CHANGES_REQUESTED** [review]
See comments

File: docs/en/hass-install-image.md (hunk 4 above)
Comment (nakata5321) [reply docs/en/hass-install-image.md:93]: Между прочим копипаст с оф сайта RPi)

Comment (nakata5321) [reply docs/en/hass-install-image.md]: https://github.com/airalab/robonomics-wiki/pull/378/commits/2b047473d5223ae7c6b2ff782d3278f476cd5304

File: docs/en/hass-install-image.md (hunk 3 above)
Comment (nakata5321) [reply docs/en/hass-install-image.md]: https://github.com/airalab/robonomics-wiki/pull/378/commits/2b047473d5223ae7c6b2ff782d3278f476cd5304

File: docs/en/hass-install-image.md (hunk 1 above)
Comment (nakata5321) [reply docs/en/hass-install-image.md]: https://github.com/airalab/robonomics-wiki/pull/378/commits/2b047473d5223ae7c6b2ff782d3278f476cd5304

File: docs/en/hass-install-image.md (hunk 4 above)
Comment (PaTara43) [reply docs/en/hass-install-image.md]: Максимально странно

**Review by PaTara43 — APPROVED** [review]

**Pull request merged**